config/201-clusterrole.yaml

```diff
@@ -132,3 +132,41 @@ rules:
   resources:
     - eventtypes
   verbs: *everything
+
+# The role is needed for the aggregated role source-observer in knative-eventing to provide readonly access to "Sources"
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: knative-gcp-pullsubscription-observer
+  labels:
+    eventing.knative.dev/release: devel
+    duck.knative.dev/source: "true"
+rules:
+  - apiGroups:
+      - "pubsub.cloud.run"
+    resources:
+      - "pullsubscriptions"
+    verbs:
+      - get
+      - list
+      - watch
+
+# The role is needed for the aggregated role source-observer in knative-eventing to provide readonly access to "Sources"
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: knative-gcp-storage-observer
+  labels:
+    eventing.knative.dev/release: devel
+    duck.knative.dev/source: "true"
+rules:
+  - apiGroups:
+      - "events.cloud.run"
```

Comment:
I think this changed in #332.

Reply:
This clusterrole never gets merged into master branch. Therefore Nacho can't see it and he didn't change it in #332